### DISCUSSION TITLE
How do you deal with packs/box sets?

### DISCUSSION THREAD

**Discussion by Matthieu-LAURENT39** [benjaminjonard/koillection#987] [General]
I mainly use Koillection to manage my manga collection, and an issue i often come across is how i should deal with box sets that contain multiple volumes.
Currently, i just add all the contained volumes as regular volumes, add a note saying it's part of a pack, and add the other contained volumes as related to the first contained volume.
As for the price, i put the full price of the pack on the first volume, and put 0 for all other volumes.

It looks something like this
![image](https://github.com/benjaminjonard/koillection/assets/91389613/1eae54ee-9057-426e-87d3-79683ff161ea)

However, i'm also considering creating an item with the pack instead (especially since they often have their own ISBN and cover), and adding the contained volumes as related items.
That would avoid having to add notes everywhere, but that would add an extra item to the list that isn't a volume.

I'm not too sure which solution is best, and there's probably no objective answer anyway.
However, i'm still curious how other users deal with this issue.

**Comment by sanmadjack** [comment]
I also would like a cleaner solution to this. Right now I'll just create the box set and individual items all as items in the collection and then relate them so I can lend them out as a set or individually.

**Comment by Matthieu-LAURENT39** [comment]
Maybe a "group"/"container" item type could be good for this?
It wouldn't count in the actual item count, but would just be used to group other items, using relations (an item would probably be only in one group, otherwise tags would be better)
And like regular items, it can have metadata and images.
As for views, maybe all "contained" items could be collapsed or expanded into it?

**Comment by azuravian** [comment]
I have something similar with Lego sets.  I'm wanting to associate each minifig with the Lego set it's a part of.  What I've done is created a Sub-collection of my Lego collection called "Minifigures".  I add the minifigs there and then add them as related to their associated set.  This keeps the primary collection as just sets (or in your case, volumes).  It does not work for lending however, as I am able to lend the set and then also lend the minifig separately.

**Comment by goinsandrew** [comment]
Not sure if this is still being worked on, but why not just make entire collections lendable? Make your box set a collection within your manga section, then you could lend out the collection (which could mark all sub items as lent), or lend just the books within (disabling lending of the parent container as its missing a portion)
